### DISCUSSION TITLE
Is there a way to disable copy on selection?

### DISCUSSION THREAD

**Discussion by trusktr** [VSCodeVim/Vim#7209] [Q&A]
Any time I select text (f.e. enter VISUAL mode), when I exit VISUAL mode, whatever I last had selected is now in my clipboard but I did not press `y` to yank it.

Unless I missed it, I do not see an option to disable this. How do I disable this, so that nothing is ever copied unless I press `y` (like regular Vim)?

**Comment by genkoph** [comment]
There is a setting called "vim.overrideCopy", set it to "false" and you're good to go. :)
